calendar-bundle/tests/EventListener/CalendarFeedListenerTest.php

```diff
@@ -0,0 +1,293 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CalendarBundle\Tests\EventListener;
+
+use Contao\CalendarBundle\CalendarEventsGenerator;
+use Contao\CalendarBundle\Event\FetchEventsForFeedEvent;
+use Contao\CalendarBundle\Event\TransformEventForFeedEvent;
+use Contao\CalendarBundle\EventListener\CalendarFeedListener;
+use Contao\CalendarEventsModel;
+use Contao\CalendarModel;
+use Contao\ContentModel;
+use Contao\Controller;
+use Contao\CoreBundle\Cache\CacheTagManager;
+use Contao\CoreBundle\Image\ImageFactoryInterface;
+use Contao\CoreBundle\InsertTag\InsertTagParser;
+use Contao\CoreBundle\Routing\ContentUrlGenerator;
+use Contao\Environment;
+use Contao\Files;
+use Contao\FilesModel;
+use Contao\Image\ImageInterface;
+use Contao\Model\Collection;
+use Contao\PageModel;
+use Contao\System;
+use Contao\TestCase\ContaoTestCase;
+use Contao\UserModel;
+use FeedIo\Feed;
+use Imagine\Gd\Imagine;
+use Imagine\Image\Box;
+use PHPUnit\Framework\Attributes\DataProvider;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Filesystem\Path;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
+
+class CalendarFeedListenerTest extends ContaoTestCase
+{
+    protected function tearDown(): void
+    {
+        parent::tearDown();
+
+        $this->resetStaticProperties([Files::class, System::class]);
+    }
+
+    #[DataProvider('featured')]
+    public function testFetchesEventsFromCalendars(string $feedFeatured, bool|null $featuredOnly): void
```

Comment:
It seems you forgot to use the second argument:

<img width="1213" alt="" src="https://github.com/user-attachments/assets/75307a91-36c4-4fe1-8e20-0b7768e0fbb5" />